libc/test/src/math/pow_test.cpp

```diff
@@ -0,0 +1,114 @@
+//===-- Unittests for pow ------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include "hdr/math_macros.h"
+#include "src/__support/FPUtil/FPBits.h"
```

Comment:
I don't see hdr/math_macros.h being used, and the `FPBits` type being used is the `using FPBits = LIBC_NAMESPACE::fputil::FPBits<T>;` from `FPTest`.

```suggestion
```

Reply:
Done.